teams/wg-wasm.toml

```diff
@@ -5,13 +5,13 @@ kind = "working-group"
 [people]
 leads = ["fitzgen"]
```

Comment:
I assume you also want to remove yourself here too?

Reply:
Done